README.md

```diff
@@ -305,6 +305,20 @@ Responses:
 `:id` Is the id of an item in the feed.
 
 
+### `GET /api/radio`
+
+> Get list of radio stations.
+
+Query parameters:
+
+* `cityName` String. If specified, returns only stations active in the given city.
+* `cityId` Integer. If specified, returns only stations active in the given city.
```

Comment:
Other end points capable of filtering results by city can do so by name or id. For consistency's sake, filtering by name should be dropped from other endpoints as well should we choose to drop it from this endpoint.